docs/dev/README.md

```diff
@@ -1,30 +1,55 @@
-# Getting started
+# zkSync Documentation
 
-::: warning Alpha testnet
+This documentation is intended to assist you in developing on zkSync. 
+It introduces the concept of zkSync, describes the zkSync technology stack, and some advanced subjects for complex applications and use cases.
 
-Welcome to zkSync 2.0 alpha testnet! Our team would be very happy to see what you can build on zkSync. We are glad for any feedback you could provide!
+Because this is a community initiative, feel free to suggest new topics, add new content, and provide examples wherever you believe they will be useful. If you're unsure how to proceed, [follow these instructions](#).
 
-Note, that the system is still in active development, so:
+## Fundamental topics
 
-- **Breaking changes are possible in the future.**
-- **Some of the updates may require regenesis**, i.e. erasing all the balances, smart contracts, and restarting the blockchain. We will make sure to communicate all regenesis beforehand! Make sure to follow our [Discord](https://discord.gg/px2aR7w).
+If this is your first time using zkSync, we recommend that you kickoff at the beginning and work your way through the documentation like you would with a book.
 
-:::
 
-**zkSync 2.0** is a [ZK rollup](./rollups.md): a trustless protocol that utilises zero-knowledge proofs to provide scalable low-cost transactions on Ethereum. All assets are stored in a single smart contract on the mainchain, while computation and storing data are performed off-chain. As all transactions are proven on Ethereum mainchain, users enjoy the same security level as in Ethereum.
+- [Intro to Rollups](./fundamentals/rollups.md) - A brief overview of rollups.
+- [zkSync overview](./fundamentals/zkSync.md) - A quick overview of zkSync technologies.
+- [zkSync testnet](./fundamentals/testnet.md) - A brief intro into zkSync testnest.
+  - [migration-testnet](./testnet/migration-to-testnet-paymaster.md) - A brief explainer on paymasters on testnet.
```

Comment:
> migration-testnet

Is it intended?

The link is also mostly about how to migrate to the paymasters for a person who was using "paying fees with ERC20 tokens" feature of the previous testnets on zkSync. I think that a brief explainer is here: https://aqwzx-zksync-v2-docs--pr97-roadmap-1-qia5fdp3.web.app/dev/developer-guides/transactions/aa.html#paymasters